src/Tutor.Core/UseCases/Management/Enrollments/LearnerGroupService.cs

```diff
@@ -32,12 +33,21 @@ public Result CreateMembers(int groupId, List<Learner> learners)
     {
         var memberships = learners.Select(l => new GroupMembership(l, groupId));
         _groupRepository.CreateBulkMemberships(memberships);
+
+        var result = _unitOfWork.Save();
+        if (result.IsFailed) return result;
+        
         return Result.Ok();
     }
 
     public Result DeleteMember(int groupId, int learnerId)
     {
-        _groupRepository.DeleteMember(groupId, learnerId);
+        var result = _groupRepository.DeleteMember(groupId, learnerId);
+        if (result.IsFailed) return result;
+
+        result = _unitOfWork.Save();
```

Comment:
To be clear, the general flow for using UoW is:

1. Invoke the correct repository method
    1. If the method in question is delete, check if the Result is failed.
2. Call UoW Save.
3. Check if its result is a failure.

1.1. stands out, as I've hinted at in one of the previous comments.